src/Chainweb/RestAPI.hs

```diff
@@ -239,14 +241,15 @@ someChainwebServer v dbs mr (HeaderStream hs) (Rosetta r) =
         <> PactAPI.somePactServers v pacts
         <> maybe mempty (Mining.someMiningServer v) mr
         <> maybe mempty (someHeaderStreamServer v) (bool Nothing cuts hs)
-        <> bool mempty (someRosettaServer v mempools) r
+        <> bool mempty (someRosettaServer v concreteMs) r
   where
     payloads = _chainwebServerPayloadDbs dbs
     blocks = _chainwebServerBlockHeaderDbs dbs
     pacts = _chainwebServerPactDbs dbs
     cuts = _chainwebServerCutDb dbs
     peers = _chainwebServerPeerDbs dbs
     mempools = _chainwebServerMempools dbs
+    concreteMs = map (second (_chainResMempool . snd)) pacts
```

Comment:
What about a more meaningful name and possibly a comment why `mempools` doesn't work here?

```suggestion
    concreteMempools = map (second (_chainResMempool . snd)) pacts
```

Reply:
Lars and I are discussing this in person.